Listener/KernelEventsSubscriber.php

```diff
@@ -54,8 +54,9 @@ public function onKernelTerminate(TerminateEvent $event): void
 
     public function onKernelException(ExceptionEvent $event): void
     {
+        $isMainRequestMethod = method_exists($event, 'isMainRequest') ? 'isMainRequest' : 'isMasterRequest';
```

Comment:
```suggestion
        $isMainRequest = method_exists($event, 'isMainRequest') ? $event->isMainRequest() : $event->isMasterRequest();
```

to avoid using the dynamic function call feature of PHP (which is probably not the best feature out there... and is hard to validate when using static analysis tools)